README.md

```diff
@@ -41,10 +41,10 @@ When you edit one of these files, run this command afterwards so that your chang
 systemctl daemon-reload
 ```
 
-To make sure the user `openfisca` is allowed to restart a service in order to redeploy it, log in as root and run
+To make sure the user `openfisca` is allowed to restart a service in order to redeploy it, log in and run:
```

Comment:
To make sure *you* are allowed ?

Reply:
Good point!

Reply:
No, actually it is the `openfisca` user.

See for example the contents of the sudoers file:

```
openfisca ALL=(ALL) NOPASSWD: /bin/systemctl restart legislation-explorer.service
```

It clearly says it is the `openfisca` user who is allowed to restart legislation explorer, not you. 

Now, you can always do it `as root`, which is another issue.

Reply:
So as sudoers we are not allowed to do it ? Can only `openfisca` do it ?

Reply:
What `sudoers` means is that you can do something on *behalf* of another user, in this case `root`. Being `sudoer` doesn't not allow you to do anything, but to "impersonate" someone else to do it. Here, we want to know if the `openfisca` user can do something or not.